src/richtext.ts

```diff
@@ -114,7 +44,7 @@ export function richTextResolver<T>(options: StoryblokRichTextOptions<T> = {} )
   const nodeResolver = (tag: string): StoryblokRichTextNodeResolver<T> => (node: StoryblokRichTextNode<T>): T => renderFn(tag, { ...node.attrs, key: `${tag}-${currentKey}` } || {}, node.children || null as any) as T
 
   const imageResolver: StoryblokRichTextNodeResolver<T> = (node: StoryblokRichTextNode<T>) => {
-    const { src, alt, ...rest } = node.attrs || {};
+    const { src, alt, title, srcset, sizes } = node.attrs || {};
```

Comment:
(Potential improvements for the future?)
I think this list of attributes is enough for now, but it made me think about it. Should we list all the possible supported `img` attributes? If so, what if a user wants to add custom attributes (such as `data-`attributes or whatever? 🤔 They cannot add them to images directly on the CMS, right? They should update the nodes themselves.

As far as I can see, I am asking this because I couldn't find either `srcset` or `sizes` in the richtext image payload.